sqlite/src/select.c

```diff
@@ -4989,6 +4992,13 @@ static int selectExpander(Walker *pWalker, Select *p){
         pTab->nCol = nCol;
       }
 #endif
+
+#if defined(SQLITE_BUILDING_FOR_COMDB2)
+      if (!(IsVirtual(pTab) || pTab->pSelect)) {
+        // add all tables referenced in query even if not needed to execute query
+        sqlite3VdbeAddTable(v, pTab);
```

Comment:
Does this mean that a table is going to be added twice, here and in `sqlite3OpenTable()` ? Though the table list is dedup'd in reqlog, this still seems a bit wasteful.